tests/tests.coffee

```diff
@@ -1,3 +1,4 @@
+{ CollectionExtensions } = require('meteor/lai:collection-extensions')
```

Comment:
Would be possible to rework this coffee script file into vanilla JS? This goes well with the spirit of a new breaking versions.